tests/integration/extenders/NotificationChannelTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Tests\integration\extenders;
+
+use Flarum\Extend\NotificationChannel;
+use Flarum\Tests\integration\RetrievesAuthorizedUsers;
+use Flarum\Tests\integration\TestCase;
+use Flarum\User\User;
+
+class NotificationChannelTest extends TestCase
+{
+    use RetrievesAuthorizedUsers;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->prepareDatabase([
+            'users' => [
+                $this->normalUser()
+            ]
+        ]);
+    }
+
+    private function add_channel()
+    {
+        $this->extend(new NotificationChannel('test'));
+    }
+
+    /**
+     * @test
+     */
+    public function can_add_notification_channel()
```

Comment:
This doesn't actually work: setUp instantiates the app before extenders are registered, meaning that the extenders don't actually need to do anything. To check that the extender registers without error, you'll need to separate out the database operations into their own method and call that explicitly after adding the extender.